### PR TITLE
Ocient explain fix

### PR DESCRIPTION
**Pull request by alexclavel-ocient**
Fixing missing whitespace in explain functionality

### CODE REVIEW

**Comment by alexclavel-ocient** [comment]
It also appears that some logo update commits got caught in this, these commits have already been merged into dbeaver:devel

**Comment by E1izabeth** [comment]
Thank you for contribution.

**Review by E1izabeth — APPROVED** [review]

**Comment by E1izabeth** [comment]
Your PR can't be built
```
Failed to execute goal org.eclipse.tycho:tycho-compiler-plugin:2.7.4:compile (default-compile) on project org.jkiss.dbeaver.ext.ocient: Compilation failure: Compilation failure: 

[ERROR] /var/jenkins_home/jobs/DBeaverDesktop/jobs/dbeaver-ce-github-checks/branches/PR-17734/workspace/dbeaver/plugins/org.jkiss.dbeaver.ext.ocient/src/org/jkiss/dbeaver/ext/ocient/model/plan/OcientExecutionPlan.java:[68] 

[ERROR] 	return String.format("explain json %s", query)

[ERROR] 	                                             ^

[ERROR] Syntax error, insert ";" to complete BlockStatements

[ERROR] 1 problem (1 error)

```

**Comment by alexclavel-ocient** [comment]
Sorry about that. Could you try again? 100% my bad

**Review by E1izabeth — APPROVED** [review]

File: plugins/org.jkiss.dbeaver.ext.ocient/src/org/jkiss/dbeaver/ext/ocient/model/plan/OcientExecutionPlan.java [hunk 1]
```diff
@@ -64,7 +64,8 @@ public String getQueryString() {
 
     @Override
     public String getPlanQueryString() {
-        return "explain json" + query;
+        // Adds "explain json" to front of query with proper spacing
+        return String.format("explain json %s", query);
```
Comment (ShadelessFox) [inline plugins/org.jkiss.dbeaver.ext.ocient/src/org/jkiss/dbeaver/ext/ocient/model/plan/OcientExecutionPlan.java]: I don't really think that formatting is required - just use concatenation here.

**Review by ShadelessFox — CHANGES_REQUESTED** [review]

File: plugins/org.jkiss.dbeaver.ext.ocient/src/org/jkiss/dbeaver/ext/ocient/model/plan/OcientExecutionPlan.java (hunk 1 above)
Comment (alexclavel-ocient) [reply plugins/org.jkiss.dbeaver.ext.ocient/src/org/jkiss/dbeaver/ext/ocient/model/plan/OcientExecutionPlan.java]: We used to have concatenation, but someone removed the trailing whitespace. This way, it is obvious that the whitespace is intended @ShadelessFox

Comment (ShadelessFox) [reply plugins/org.jkiss.dbeaver.ext.ocient/src/org/jkiss/dbeaver/ext/ocient/model/plan/OcientExecutionPlan.java]: It's obvious the other way around, too.

Comment (alexclavel-ocient) [reply plugins/org.jkiss.dbeaver.ext.ocient/src/org/jkiss/dbeaver/ext/ocient/model/plan/OcientExecutionPlan.java:68]: wdym? it wasn't obvious enough clearly if it was removed unintentionally. See https://github.com/Xeograph/dbeaver/commit/41745c344fdfc07bc36547342352d765524c758f

**Review by LonwoLonwo — COMMENTED** [review]
Just do it simple

`"explain json " + query;`

**Review by LonwoLonwo — APPROVED** [review]

File: plugins/org.jkiss.dbeaver.ext.ocient/src/org/jkiss/dbeaver/ext/ocient/model/plan/OcientExecutionPlan.java [hunk 2]
```diff
@@ -65,7 +65,7 @@ public String getQueryString() {
     @Override
     public String getPlanQueryString() {
         // Adds "explain json" to front of query with proper spacing
-        return String.format("explain json %s", query);
+        return "explain json" + " " + query;
```
Comment (ShadelessFox) [inline plugins/org.jkiss.dbeaver.ext.ocient/src/org/jkiss/dbeaver/ext/ocient/model/plan/OcientExecutionPlan.java]: Please, just a single space:

```suggestion
        return "explain json " + query;
```

It's really obvious that the space is essential here, no need to make it even more obvious and smelly.

**Review by ShadelessFox — CHANGES_REQUESTED** [review]

**Review by ShadelessFox — APPROVED** [review]

**Pull request merged**